integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -1718,6 +1733,13 @@ def do_it(spark):
     assert_gpu_and_cpu_are_equal_collect(do_it,
         conf={'spark.sql.ansi.enabled': 'true'})
 
+@ignore_order(local=True)
+@pytest.mark.parametrize('data_gen', [ArrayGen(sub_gen, nullable=False) for sub_gen in all_basic_gens + decimal_gens])
+def test_grpby_array_agg_on_func(data_gen):
```

Comment:
I am not 100% sure of the point of this test vs the other tests that you already added with  `test_hash_grpby_sum_count_action`

If we want coverage of more aggregations, then why not include these types in other already existing tests for those too instead?